### DISCUSSION TITLE
Table doesn't exist

### DISCUSSION THREAD

**Discussion by xhorizont** [joomengine/Joomla-Component-Builder#772] [Extention Issues]
Hi,

I have made a simple first component and after compile and install, I go to View or Add asset and get Table 'dev2server.#__itam_asset' doesn't exist

Have I missed something

**Comment by ohrionmartin** [comment]
Hi @xhorizont 
I can't seem to reproduce this on my development environment, watch the [Hello World](https://youtu.be/1KBBtQUxMTc) tutorial. It could be that you are probably just missing something, I hope it helps..

**Reply by xhorizont** [reply]
I forgot the admin view :)

**Comment by cpaschen** [comment]
Just a thought ... I have this same thing happen every once in a while because  I forget to add the admin view to the component.
As noted, you might want to go back and review the tutorial, but also you could double check to make sure you added the admin view to the component specifically.

**Reply by xhorizont** [reply]
I forgot the admin view :)

**Comment by xhorizont** [comment]
Now I have another problem. I compiled 3 test components and I can add records but button for view record doesn't work

**Comment by cpaschen** [comment]
You'll definitely want to go through the videos as these are basic issues.
What I did when I first started using JCB was to create a quick check-list of what to (and not forget) and in what order. That helped me a LOT.
As for the view record button, I think there's only one place that this is set - in the component's admin views listing.
